xrpl/models/base_model.py

```diff
@@ -54,7 +72,18 @@ def _key_to_json(field: str) -> str:
         1. 'TransactionType' becomes 'transaction_type'
         2. 'value' remains 'value'
         3. 'URI' becomes 'uri'
+
+        This function accepts inputs in camelCase only
+
+    Raises:
+        XRPLModelException: If the input is invalid
     """
+    if is_snake_case(field):
+        raise XRPLModelException(
+            "_key_to_json accepts input strings in the camelCase format only, "
```

Comment:
This isn't true based on your comment. It accepts keys in full lowercase as well.

Reply:
thanks. I have removed this function 👍

Reply:
fixed in [7405ba3](https://github.com/XRPLF/xrpl-py/pull/694/commits/7405ba3e197f840519629679d2da418639d5e0d6)